.github/workflows/release.yml

```diff
@@ -85,92 +85,22 @@ jobs:
 
   build-and-sign:
     permissions:
-      # cosign uses the GitHub OIDC token
       id-token: write
+      contents: write
```

Comment:
What was content write used for? I don't see anything that introduced that

Reply:
That was added for testing purposes. Good catch! will remove that